tests/components/webostv/test_config_flow.py

```diff
@@ -73,14 +53,14 @@ async def test_form(hass: HomeAssistant, client) -> None:
     assert result["type"] is FlowResultType.FORM
     assert result["step_id"] == "pairing"
 
-    result = await hass.config_entries.flow.async_configure(
+    result2 = await hass.config_entries.flow.async_configure(
```

Comment:
(personal opinion: we don't use `result` anymore, so might as well just overwrite)

Reply:
Agree, I had doubts which way to go and aligned all tests, but we don't need it